src/xdocs/config_modifier.xml

```diff
@@ -446,13 +445,21 @@ public interface RoadFeature {
             <code>strictfp</code>
           </li>
         </ol>
+        <p>
+          In additional, modifiers are checked to ensure all annotations are
+          declared before all other modifiers.
+        </p>
+        <p>
+          Rationale: Code is easier to read if everybody follows a standard.
+        </p>
+      </subsection>
 
+      <subsection name="Notes" id="ModifierOrder_Notes">
```

Comment:
Why is this now a separate section?

Reply:
It starts with `ATTENTION` and sounds like an important note. 
Am i wrong? I can revert  this.

Reply:
I am not saying your wrong, but it seems like we haven't done this before now.
https://checkstyle.sourceforge.io/config_coding.html#DeclarationOrder_Description
This still has attention on the bottom.

"Attention" is something that should be made to stand out and known, like a bug or a limitation. `Notes` sounds like the wrong section for something of this importance. What if we do add extra things of smaller importance to the same notes, it would be back to as it was before.

@romani Do you have any opinion on this?

Reply:
I do not have strong opinion on this, I am ok in either way.

Reply:
In such cases, when non have strong position, request of reviewer should be satisfied, as it good view from aside. Other people might find it same strange, as reviewer did.

Reply:
reverted back to one section